cloudfoundry/resource_cf_app_test.go

```diff
@@ -1321,6 +1353,78 @@ func TestAccApp_dockerApp(t *testing.T) {
 		})
 }
 
+func TestAccApp_url_docker_switch(t *testing.T) {
```

Comment:
Should we also have a test for switching in the other direction?

Reply:
+1, I have the same question